src/track_api.c

```diff
@@ -156,6 +156,13 @@ void tracker_bit_sync_update(tracker_context_t *context, u32 int_ms,
   tracker_internal_data_t *internal_data;
   tracker_internal_context_resolve(context, &channel_info, &internal_data);
 
+  /* L2C bit sync is known once we start tracking it since
```

Comment:
It would be cleaner and more flexible to add a `bit_sync_set()` function in bit_sync.c (libswiftnav) which simply sets `bit_phase_ref` to the specified value. Then add a function to this file like `tracker_bit_sync_set()` which calls `bit_sync_set()`.

With those changes you should be able to leave `tracker_bit_sync_update()` unchanged and get nav bit output.